research/conditional/utils/model_utils.py

```diff
@@ -71,7 +71,7 @@ def chungized_llm_loss(
 
     def make_custom_forward():
         def custom_forward(*inputs):
-            output = model.head(inputs[0])
+            output = model.full_model.head(inputs[0])
             with torch.autocast(device_type="cuda", enabled=False, dtype=torch.float16):
                 gt = inputs[1]
                 mask = inputs[2]
```

Comment:
this is major ugly, could we have sth like `x, gt, mask = inputs`? If not in this PR, maybe an issue with this?

Reply:
Orthogonal